rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -68,12 +144,49 @@ components:
     Accepted:
       description: The transformation request was accepted for processing.
       headers:
-        'Content-Version':
+        Content-Version:
+          $ref: '#/components/headers/ContentVersion'
+        Location:
+          $ref: '#/components/headers/Location'
+    TransformationResponse:
```

Comment:
✏️ This currently doesn't get generated as a class.
```java
default ResponseEntity<MetadataInformationResponse> poll(...)
```
I suggest inlining this and renaming `MetadataInformationResponse` to `TransformStatus` or `PollResponse`.